### PR TITLE
Update privacy documentation

### PR DESCRIPTION
**Pull request by andrewmaier**
Starting to address #312 per many ongoing conversations with Richard Speidel and the GSA Privacy Office. We're working on updating to the policy that's linked to from this commit. See [this Google Doc](https://docs.google.com/document/d/1jFwb_nUVln6Lbr8tIVNf1N0_BKz371B5E73U5niQVOM/edit#heading=h.5gvoa7h7px5c) to get a sense of the ongoing conversation.

### CODE REVIEW

**Comment by andrewmaier** [comment]
Hmm. Apparently that PII page is well loved (or at least thoroughly linked to) inside of this guide ;)

**Comment by andrewmaier** [comment]
For those playing along at home, the new version looks like this. I would like Richard to review this draft.

image

**Comment by andrewmaier** [comment]
@aidanfeldman Do we need to define (and maintain a definition for) PII in this guide?

**Comment by andrewmaier** [comment]
Updated the guide to remove links to a specific page that defines PII. 

I've also removed pages/links for SORNs and PIAs, as GSA's Privacy Office is currently considering updating its approach to both. In the interim, the Privacy page, which links to the privacy threshold analysis (PTA), will serve as a sufficient entry point for all things related to privacy compliance.

File: _pages/privacy.md [hunk 1]
```diff
@@ -2,10 +2,22 @@
...
+    <p>It&rsquo;s important to note that the Google Form mentioned in step one is designed to prompt a conversation, and that this conversation may ultimate affect the design of the product or service in question. This conversation might also result in the creation of the following privacy-related documentation:</p>
+    <ul>
+      <li><a href="https://docs.google.com/a/gsa.gov/document/d/1CcVLPNNra1WCGqHewK2ojQ_ysHcGxmJ1IlsCo9pAiSU/edit?usp=drive_web">Privacy Act Notices</a></li>
+      <li><a href="https://docs.google.com/document/d/1hDIDvcTv-zplMBl_uQbg7GQ5uwrC-2Jr8xg4D073q5Y/edit">System of Records Notices (SORNs)</a></li>
+      <li><a href="https://docs.google.com/document/d/1o6FtuHPW41ZLkjXnANjZ60-sUojDexh3r3afm1yE9b8/edit#heading=h.s85zskw81fkz">Privacy Impact Assessments (PIAs)</a></li>
+    </ul>
+  </li>
+  <li>As you complete and receive sign off on your product or service&rsquo;s privacy-related documentation, the Privacy Office may work with you to plan for the creation of any necessary role-based training for the staff who will operate the product or service. This ensures privacy compliance in perpetuity.</li>
+</ol>
```
Comment (afeld) [inline _pages/privacy.md]: Can we do this in Markdown instead, for easier editing and consistency with the rest of the repository?

File: _pages/privacy.md [hunk 2]
```diff
@@ -2,10 +2,22 @@
...
 
-* [Privacy Impact Assessment (PIA)](pia/)
-* [Personally Identifiable Information (PII)](pii/)
-* [System of Records Notice (SORN)](sorn/)
+<p>As someone interested in shipping a product or service, you&rsquo;ll need to first consider <strong>the agency</strong> at which the product or service will be authorized to operate. If that agency is someone <i>other</i> than GSA, please stop reading this guide and contact the Privacy Office at the authorizing agency. Otherwise, take the following steps:</p>
 
-For help, reach out to [#wg-privacy](https://gsa-tts.slack.com/messages/C4F2HMGBE/).
+<ol>
+  <li>
+    <a href="https://gsa.gov/portal/getMediaData?mediaId=199855">GSA Policy 1878.2A CIO P Conducting Privacy Impact Assessments (PIAs)</a> requires that each of the GSA&rsquo;s information systems, either planned or under development, be subject to an annual privacy threshold analysis. Please <a href="https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLScrhrrzgS-ZcOpS37YzLdmVXkwX_3dBefZ1t39oKcgdVepmuQ/viewform">fill out and submit this Google Form</a> to get started.
```
Comment (afeld) [inline _pages/privacy.md]: Seeing as the "fill out and submit this Google Form" part is the major call to action on this page, I would bold it, and/or put it as a TL;DR at the top.

File: _pages/privacy.md [hunk 3]
```diff
@@ -2,10 +2,22 @@
...
+  </li>
+  <li>
+    <p>Once they&rsquo;ve had a chance to review your information system, GSA&rsquo;s Privacy Office will reach out to help you collaboratively identify, discuss, and evaluate privacy risks, and to determine the ways in which they might be mitigated. Generally speaking, a system&rsquo;s privacy risk stems from to the degree to which that system and its associated business processes collect and use personally identifiable information (PII).</p>
```
Comment (afeld) [inline _pages/privacy.md]: Mind moving this second sentence out of the list, to keep the list focused on the review process? Could similarly slim down the other bullet points as well.

File: _pages/privacy/pii.md [hunk 4]
```diff
@@ -1,50 +0,0 @@
...
-You must inform people from whom you wish to collect PII of the reason you are requesting their information; how you will use their information; and what the consequences are, if any, of their refusal to provide the information you've requested. Implicit in this requirement is the notion of informed consent. 
-
-Privacy Act Statements must be accessible where the PII is collected; for example, on the webpage where the user inputs information. When drafting a Privacy Act Statement, make sure to include the following elements:
-
-* The legal authority for collecting the information – statute, executive order, regulation.
-* The purpose(s) for collecting the information and how you will use it.
-* A reference to the SORN that covers your system; and
-* Whether providing the information is mandatory or voluntary and what the effects of not providing the information are – for example the loss or denial of a privilege, benefit, or entitlement.
-
-Once you've drafted the statement, submit it for review to the Privacy Office.
```
Comment (afeld) [inline _pages/privacy/pii.md]: Maybe want to mention Privacy Act Statements in the expanded page?

**Comment by rspeidel** [comment]
Thanks Andrew for drawing this up and Aidan for the feedback. It looks great generally and I don't see any problem moving the PTA form up.  
Please reference the definition of PII in OMB A-130.  I am trying to consolidate all GSA to that:  Personally Identifiable Information (PII) is information that can be used to distinguish or trace an individual’s identity, either alone or when combined with other information that is linked or linkable to a specific individual.  https://obamawhitehouse.archives.gov/sites/default/files/omb/assets/OMB/circulars/a130/a130revised.pdf

File: _pages/privacy.md (hunk 1 above)
Comment (andrewmaier) [inline _pages/privacy.md]: Today I learned how to nest lists in Markdown

**Comment by andrewmaier** [comment]
Just seeing the comment from @rspeidel. I'll update the definition and link to A130.

File: _pages/privacy/pii.md (hunk 4 above)
Comment (andrewmaier) [inline _pages/privacy/pii.md]: This page no longer exists.

**Comment by andrewmaier** [comment]
Preview of changes thus far
image

File: _pages/privacy/pii.md (hunk 4 above)
Comment (afeld) [inline _pages/privacy/pii.md]: Right...I'm saying that Privacy Act Statements still seem like they are relevant to us, so I think they still need to be mentioned somewhere.

File: _pages/privacy/pii.md [hunk 5]
```diff
@@ -1,50 +0,0 @@
...
-
-* Citizenship or immigration status
-* Ethnic or religious affiliation
-* Mother's maiden name
-* Last 4 digits of SSN
-* Medical information
-* Sexual orientation
-* Account passwords
-* Criminal history
-* Date of birth
```
Comment (afeld) [inline _pages/privacy/pii.md]: I still feel that including these examples is helpful for teams to understand the types of information that the formal definition is referring to. Unless @rspeidel objects, of course.

**Comment by afeld** [comment]
Made some small tweaks - hope they're ok with you!

File: _pages/privacy/pii.md (hunk 4 above)
Comment (andrewmaier) [inline _pages/privacy/pii.md]: Ohh. Well, that's — Privacy Act Statements only apply to forms that collect PII (as far as I'm aware), which is something that should come up in the privacy threshold analysis process.

Comment (andrewmaier) [inline _pages/privacy/pii.md]: @rspeidel ^^

Comment (afeld) [inline _pages/privacy/pii.md]: My thinking was to include something as simple as

> If your form is collecting PII, you may need to add a Privacy Act Statement.

I hear your point though 👍

File: _pages/privacy/pii.md (hunk 5 above)
Comment (afeld) [inline _pages/privacy/pii.md]: I think this is the only blocker to merging.

**Comment by andrewmaier** [comment]
Preview of recent commit
image

**Pull request merged**